pages/sign-in/index.js

```diff
@@ -0,0 +1,9 @@
+const SignIn = () => {
+  return (
+    <div>
+      <h1>Sign-in</h1>
```

Comment:
We have a Heading1 component. Do we want to use this here in the page instead of h1?

Reply:
Good call out. Updated.